packages/storage-postgres/migrations/0053_space_geo_location.sql

```
@@ -0,0 +1,34 @@
+--> statement-breakpoint
+ALTER TABLE "spaces" ADD CONSTRAINT "spaces_location_source_enum" CHECK (
+  "location_source" IS NULL
+  OR "location_source" IN ('geocode', 'manual', 'map_click')
+);
+--> statement-breakpoint
+ALTER TABLE "spaces" ADD CONSTRAINT "spaces_geo_not_null_island" CHECK (
+  NOT ("latitude" = 0 AND "longitude" = 0)
+);
+--> statement-breakpoint
+CREATE INDEX "spaces_geo_idx" ON "spaces" ("latitude", "longitude")
+WHERE "latitude" IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)